web/pdf_sidebar.js

```diff
@@ -53,23 +53,17 @@ var SidebarView = {
  *   for documents containing outline/attachments. The default value is `false`.
  */
 
-/**
- * @class
- */
-var PDFSidebar = (function PDFSidebarClosure() {
+class PDFSidebar {
   /**
-   * @constructs PDFSidebar
    * @param {PDFSidebarOptions} options
    */
-  function PDFSidebar(options) {
+  constructor(options) {
     this.isOpen = false;
     this.active = SidebarView.THUMBS;
     this.isInitialViewSet = false;
 
-    /**
-     * Callback used when the sidebar has been opened/closed, to ensure that
-     * the viewers (PDFViewer/PDFThumbnailViewer) are updated correctly.
-     */
+    // Callback used when the sidebar has been opened/closed, to ensure that
+    // the viewers (PDFViewer/PDFThumbnailViewer) are updated correctly.
```

Comment:
Nit: I followed the same pattern as in [`PDFDocumentLoadingTask`](https://github.com/mozilla/pdf.js/blob/ece24dfe61d0cd6fe7a8b23728336efd4033fda7/src/display/api.js#L314-L351) when adding this comment, so unless you object violently my preference would be that we keep the formatting intact here :-)